bin/zopen-install

```diff
@@ -383,11 +452,21 @@ handlePackageInstall()
       rm -rf "${baseinstalldir}/${installdirname}"
     fi
 
-    if ! runLogProgress "pax -rf ${pax} -p p ${paxredirect} ${redirectToDevNull}" "Expanding ${pax}" "Expanded"; then
-      printWarning "Errors unpaxing, package directory state unknown."
-      printInfo "Use zopen alt to select previous version."
-      continue
+    extractPax=true
+    metadataVersion=$(jq -r '.version_scheme' "${metadataFile}" 2>/dev/null)
+    is_greater=$(echo "$metadataVersion > 0.1" | bc -l)
+
+    if [ "$is_greater" -eq 1 ] && [ "$skipverify" = "false" ]; then
+           verifySignatureOfPax
     fi
+    if [ "$extractPax" = true ]; then
```

Comment:
Is extractPax needed anymore?

Reply:
Yes it is set to false under certain conditions in verifySignatureOfPax

Reply:
Is there a case without the printError?